chainercv/links/model/faster_rcnn/faster_rcnn_train_chain.py

```diff
@@ -0,0 +1,157 @@
+import numpy as np
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+
+from chainercv.links.model.faster_rcnn.utils.anchor_target_creator import \
+    AnchorTargetCreator
+from chainercv.links.model.faster_rcnn.utils.proposal_target_creator import \
+    ProposalTargetCreator
+
+
+class FasterRCNNTrainChain(chainer.Chain):
+
+    """Calculate losses for Faster R-CNN and report them.
+
+    This is used to train Faster R-CNN in the joint training scheme [#]_.
+
+    .. [#] Shaoqing Ren, Kaiming He, Ross Girshick, Jian Sun. \
+    Faster R-CNN: Towards Real-Time Object Detection with \
+    Region Proposal Networks. NIPS 2015.
+
+    Args:
+        faster_rcnn (~chainercv.links.model.faster_rcnn.FasterRCNN):
+            A Faster R-CNN model that is going to be trained.
+        rpn_sigma (float): Sigma parameter for the localization loss
+            of Region Proposal Network (RPN).
```

Comment:
How about adding a note about the default value? Is it the value used in the original paper?